pkg/operator/nfs/spec.go

```diff
@@ -84,6 +84,7 @@ func newStatefulSetForNFSServer(cr *nfsv1alpha1.NFSServer) *appsv1.StatefulSet {
 					Labels:    newLabels(cr),
 				},
 				Spec: corev1.PodSpec{
+					ServiceAccountName: "rook-nfs-server",
```

Comment:
Do you think this is the right approach to hard code the service account name within the operator?  If a user modifies the example yamls to use a different service account name besides "rook-nfs-server", then this operator will be telling the pods to use a service account that doesn't exist, right?

it looks like we were taking in the service account name via a spec field on the NFSServer object...should we just use that field still, so that it's entirely within the user's control to keep in sync the name of the service account that is **created** and the name of the service account that is **used**?

Reply:
In [this comment](https://github.com/rook/rook/pull/5785#discussion_r452996362) I pointed out that in the ceph operator we haven't found it necessary to configure the service account name, so you can blame me if you disagree. :)